### DISCUSSION TITLE
Confirmation on the client side

### DISCUSSION THREAD

**Discussion by cedrticgayte** [amidaware/tacticalrmm#2070] [Q&A]
Hello,

I would like to add a confirmation on the client side so that users can authorize or not the takeover.
I would like to respect the French regulation in terms of GDPR

regards

**Comment by wh1te909** [comment]
https://docs.tacticalrmm.com/tipsntricks/#enable-remote-control-options

**Comment by cedrticgayte** [comment]
Thanks a lot
